README.md

```diff
@@ -1,23 +1,31 @@
-# Dogecoin Core [DOGE, Ð]
-==========================
+<p align="center">
+<img src="http://static.tumblr.com/ppdj5y9/Ae9mxmxtp/300coin.png"
+</p>
+<h1 align="center">Dogecoin Core [DOGE, Ð]</h1>
 
-![Dogecoin](https://static.tumblr.com/ppdj5y9/Ae9mxmxtp/300coin.png)
+[![Build Status](https://travis-ci.com/dogecoin/dogecoin.svg?branch=master)](https://travis-ci.com/dogecoin/dogecoin) [![tip for next commit](https://tip4commit.com/projects/702.svg)](https://tip4commit.com/github/dogecoin/dogecoin)
 
-[![Build Status](https://travis-ci.org/dogecoin/dogecoin.svg?branch=1.14-dev)](https://travis-ci.org/dogecoin/dogecoin) [![tip for next commit](https://tip4commit.com/projects/702.svg)](https://tip4commit.com/github/dogecoin/dogecoin)
+## What is Dogecoin? – Such coin 🤔  
 
-## What is Dogecoin? – Such coin
-Dogecoin is a cryptocurrency like Bitcoin, although it does not use SHA256 as its proof of work (POW). Taking development cues from Tenebrix and Litecoin, Dogecoin currently employs a simplified variant of scrypt.
+Dogecoin (styled as DOGE or Ð) is a cryptocurrency like Bitcoin, although it does not use SHA256 as its proof of work (POW). Taking development cues from Tenebrix and Litecoin, Dogecoin currently employs a simplified variant of scrypt.
+- **Website:** [dogecoin.com.](https://dogecoin.com)
+- **Get started with doge here:** [dogecoin.com/getting-started.](https://dogecoin.com/getting-started.)
```

Comment:
That section is outdated right now, the main page actually has better instructions.

Reply:
Do you want me to use the section from the Master branch?

Reply:
Removed the getting started in favour of the main dogecoin.com website.